charts/ChartConfig.tsx

```diff
@@ -223,7 +233,7 @@ export class ChartConfig {
         if (!firstVar) return formatYear
 
         return firstVar.display.yearIsDay
-            ? (day: number) => formatDay(day, firstVar.display.zeroDay)
+            ? (day: number) => formatDay(day, EPOCH_DATE)
```

Comment:
Will the charts show the years as increments against this one?

Also, if that second param is now always a constant maybe we could remove.

Reply:
> Will the charts show the years as increments against this one?

Yes all variables will have the days ("years") shifted to start from `EPOCH_DATE`, if I understand your question.

> Also, if that second param is now always a constant maybe we could remove.

Just removed it!